src/test/gen_list.ts

```diff
@@ -0,0 +1,79 @@
+const fs = require('fs')
+import { getOpenseaCollectionAddresses } from "../../script/opesea_contrats"
+
+import {
+    Ethereum,
+    getChainAssetsPath,
+    readDirSync,
+    readFileSync,
+    isChainWhitelistExistSync,
+    isChainBlacklistExistSync,
+    getChainWhitelistPath,
+    getChainBlacklistPath,
+    writeFileSync,
+    sortDesc,
+    getUnique,
+    mapList
+} from './helpers'
+
+// Current chains where assets tokens supported by Trust Wallet
+// TODO auto create list based on assets folder presence
+const chainsSupportedWhiteAndBlacklist = [
+    'binance',
```

Comment:
use `@trustwallet/types` lib to pull coin types

Reply:
https://github.com/trustwallet/assets/pull/502/commits/9e8ab2a3deb361b38deca959c91fbd5e46117478